configs/openmrs_config/ampathforms/NCD_Consultation.json

```diff
@@ -32,7 +32,33 @@
 							},
 							"type": "obs",
 							"validators": []
-						}
+						},
+						{
+						"id": "TBSymptoms",
+						"hide": {
+							"hideWhenExpression": "isEmpty(TBsx) ||  TBsx !== '1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'"
```

Comment:
@kazlaw could you throw a light on what this logic is performing?

Reply:
Hi @kdaud , So when the question loads by default that section is hidden. When you select Yes the section appears when you select No the section gets hidden.  A very helpful link is [here ](https://ampath-forms.vercel.app/docs/conditional-rendering) about Ampath Forms